installer/build/container/push.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+# Copyright 2018 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -eu
+
+OVA_REV=$(git rev-parse --verify --short=8 HEAD)
+IMAGE="cluster-api-ova-build"
+REPO="docker.io/frapposelli/"
```

Comment:
We now have a repo for the provider container if you want to use the same repo.  gcr.io/cnx-cluster-api (part of VMW's gcr account).  You just need the GCR key file, which I can provide.

Reply:
let's follow up offline with that and I'll amend once it's pushed

Reply:
pushed and amended